toolkit/tools/scheduler/scheduler.go

```diff
@@ -387,6 +388,19 @@ func buildAllNodes(stopOnFailure, isGraphOptimized, canUseCache bool, packagesTo
 					}
 				}
 
+				if res.Node.Type == pkggraph.TypeBuild && res.WasDelta {
+					logger.Log.Tracef("This is a delta result, update the graph with the new delta files for '%v'.", res.Node)
+					// We will need to update the graph with paths to any delta files that were actually rebuilt.
+					err = setAssociatedDeltaPaths(res, res.BuiltFiles, pkgGraph, graphMutex)
+					if err != nil {
+						// Failures to manipulate the graph are fatal.
```

Comment:
Nit-pick: I think we don't need that comment, since the `stopBuilding = true` line makes our intentions very clear.;)

Reply:
Done